src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -1524,9 +1565,36 @@ def post_process(self, r):
                 while r["properties"]["provisioningState"].lower() == "inprogress":
                     r = self.client.show(self.cmd, self.get_argument_resource_group_name(), self.get_argument_name())
                     time.sleep(1)
-                linker_client.linker.begin_create_or_update(resource_uri=r["id"],
-                                                            parameters=item["parameters"],
-                                                            linker_name=item["linker_name"]).result()
+                service_connectors_def_list, service_bindings_def_list = parse_service_bindings(
+                    self.cmd, self.get_argument_service_bindings(), self.get_argument_resource_group_name(), self.get_argument_name())
+                self.set_argument_service_connectors_def_list(
+                    service_connectors_def_list)
+
+                # Case: Kafka on confluent cloud with two linker operations.
+                if any(len(item["parameters"]) ==
+                       2 for item in service_connectors_def_list):
+                    service_connectors_def_list = update_connectors_with_two_parameters(
+                        item, service_connectors_def_list.copy(), r["id"])
+
+                # Validate uniqueness across all bindings
+                check_bindings_and_raise_error(
```

Comment:
For `check_unique_bindings`, the `ContainerAppPreviewCreateDecorator` execute the `check_unique_bindings` in `construct_payload`.
The `ContainerAppPreviewUpdateDecorator` didn't have `check_unique_bindings` before. 
I recommend add `check_unique_bindings` in the `construct_payload` of `ContainerAppPreviewUpdateDecorator`， keep consistent with `ContainerAppPreviewCreateDecorator`.

Reply:
refer to the comment on 2 classes of bindings. also if you have any more suggestions on the logic implementation, please share away. i wanted to streamline the logic n make it as non-repetitive as possible and so any suggestions to that end, i welcome.